### ISSUE TITLE
Cannot change the message font size

### ISSUE THREAD

**Issue by hollisticated-horse** [syphon-org/syphon#554]
It is currently impossible to change the message font size. Nothing happens when tapping on the setting, unlike changing the general font size. 

I don't know if it's by design, bit it's a bit strange. 

Also, it would be nice to have a bit of a larger range of font sizes. Especially larger ones.

**Comment by hollisticated-horse** [comment]
Closing due to not using the software any more, and inactivity here.
